tests/test_dragon_installer.py

```diff
@@ -187,26 +190,39 @@ def test_retrieve_cached(
 @pytest.mark.parametrize(
     "dragon_pin,pyv,is_found,is_crayex",
     [
+        # Dragon V0.8
         pytest.param("0.8", "py3.8", False, False, id="0.8,python 3.8"),
         pytest.param("0.8", "py3.9", True, False, id="0.8,python 3.9"),
         pytest.param("0.8", "py3.10", True, False, id="0.8,python 3.10"),
         pytest.param("0.8", "py3.11", True, False, id="0.8,python 3.11"),
         pytest.param("0.8", "py3.12", False, False, id="0.8,python 3.12"),
-        pytest.param("0.8", "py3.8", False, True, id="0.8,python 3.8,CrayEX"),
-        pytest.param("0.8", "py3.9", True, True, id="0.8,python 3.9,CrayEX"),
-        pytest.param("0.8", "py3.10", True, True, id="0.8,python 3.10,CrayEX"),
-        pytest.param("0.8", "py3.11", True, True, id="0.8,python 3.11,CrayEX"),
-        pytest.param("0.8", "py3.12", False, True, id="0.8,python 3.12,CrayEX"),
+        pytest.param("0.8", "py3.8", False, False, id="0.8,python 3.8,CrayEX"),
+        pytest.param("0.8", "py3.9", True, False, id="0.8,python 3.9,CrayEX"),
+        pytest.param("0.8", "py3.10", True, False, id="0.8,python 3.10,CrayEX"),
+        pytest.param("0.8", "py3.11", True, False, id="0.8,python 3.11,CrayEX"),
+        pytest.param("0.8", "py3.12", False, False, id="0.8,python 3.12,CrayEX"),
+        # Dragon V0.9
         pytest.param("0.9", "py3.8", False, False, id="0.9,python 3.8"),
         pytest.param("0.9", "py3.9", True, False, id="0.9,python 3.9"),
         pytest.param("0.9", "py3.10", True, False, id="0.9,python 3.10"),
         pytest.param("0.9", "py3.11", True, False, id="0.9,python 3.11"),
         pytest.param("0.9", "py3.12", False, False, id="0.9,python 3.12"),
-        pytest.param("0.9", "py3.8", False, True, id="0.9,python 3.8,CrayEX"),
-        pytest.param("0.9", "py3.9", True, True, id="0.9,python 3.9,CrayEX"),
-        pytest.param("0.9", "py3.10", True, True, id="0.9,python 3.10,CrayEX"),
-        pytest.param("0.9", "py3.11", True, True, id="0.9,python 3.11,CrayEX"),
-        pytest.param("0.9", "py3.12", False, True, id="0.9,python 3.12,CrayEX"),
+        pytest.param("0.9", "py3.8", False, False, id="0.9,python 3.8,CrayEX"),
+        pytest.param("0.9", "py3.9", True, False, id="0.9,python 3.9,CrayEX"),
+        pytest.param("0.9", "py3.10", True, False, id="0.9,python 3.10,CrayEX"),
+        pytest.param("0.9", "py3.11", True, False, id="0.9,python 3.11,CrayEX"),
+        pytest.param("0.9", "py3.12", False, False, id="0.9,python 3.12,CrayEX"),
```

Comment:
It appears that these tests no longer do what they were intended to. I think 210-214 should all have true as `is_crayex` since 205-209 tested the !HSN case. Otherwise, the second set of parameterizations is the same as the first half?

Reply:
Consider updating the verbiage since you're already here. Swap `CrayEX` with `HSN` in test names and in the parameterization (is_crayex becomes is_hsn)

Reply:
Whoops! I flipped the `is_crayex` flag when I meant to flip the `is_found` flag... 🤦 Let me fix that rq